css/style.css

```diff
@@ -0,0 +1,281 @@
+/* Reset */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Main Styling */
+html,
+body {
+    font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+    line-height: 1.7em;
+}
+
+a {

```

Comment:
overwriting the default `a` works fine however I recommend doing something like `.text-white` that way you can opt into the color for dark background vs light ones.